lib/PutItemBuilder.js

```diff
@@ -12,9 +12,17 @@ var typeUtil = require('./typeUtil')
  */
 function PutItemBuilder(options) {
   Builder.call(this, options)
+
+  /** @private {string} */
+  this._returnValues = 'ALL_OLD'
 }
 require('util').inherits(PutItemBuilder, Builder)
 
+PutItemBuilder.prototype.setReturnValues = function (val) {
+  this._returnValues = val
```

Comment:
I think that's a good idea. Do you think we should have an enum mapping somewhere?